lms/static/js/commerce/views/receipt_view.js

```diff
@@ -0,0 +1,198 @@
+/**
+ * View for the receipt page.
+ */
+var edx = edx || {};
+
+(function ($, _, _s, Backbone) {
+    'use strict';
+
+    edx.commerce = edx.commerce || {};
+
+    edx.commerce.ReceiptView = Backbone.View.extend({
+        useEcommerceApi: true,
+
+        initialize: function () {
+            this.useEcommerceApi = !!($.url('?basket_id'));
+            _.bindAll(this, 'renderReceipt', 'renderError');
+
+            /* Mix non-conflicting functions from underscore.string (all but include, contains, and reverse) into
+             * the Underscore namespace.
+            */
+            _.mixin( _s.exports() );
+        },
+
+        renderReceipt: function(data){
+            var templateHtml = $( "#receipt-tpl" ).html(),
+                context = {
+                    platformName: this.$el.data('platform-name'),
+                    verified: this.$el.data('verified').toLowerCase() === 'true'
+                };
+
+            // Add the receipt info to the template context
+            _.extend(context, {
+                receipt: this.receiptContext(data),
+                courseKey: this.getOrderCourseKey(data)
+            });
+
+            this.$el.html( _.template( templateHtml, context ) );
+
+            this.trackLinks();
+        },
+
+        renderError: function() {
+            // Display an error
+            $('#error-container').show();
```

Comment:
jQuery's hide and show function are not performant, please replace with addClass / showClass alternatives to give users the optimal experience.